src/averageAges.js

```diff
@@ -15,12 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const filteredMen = century
+    ? people.filter(
+      person => Math.ceil(person.died / 100) === century
+      && person.sex === 'm')
+    : people.filter(person => person.sex === 'm');
```

Comment:
You can use 1 `filter` here and expand his conditions. Like you did in `calculateAverageAgeDiff`.

```suggestion
  const filteredMen = people.filter(person => (
    person.sex === 'm' && (century
    ? Math.ceil(person.died / 100) === century
    : true)
  );
```